### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by easy-are**
trying to add the preevy tool

### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Found this cool tool called preevy for provisioning self hosted preview environments. I thought others could benefit from it so I am suggesting it to a few relevant lists
---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by ahpook** [comment]
@easy-are thanks for the contribution. New items need to be appended to the bottom of the list, however. Please fix.

**Review by ahpook — CHANGES_REQUESTED** [review]
The tests failed because new items need to be at the bottom of the list, not the top. This is to prevent people from artificially ranking their tools to the top of the page. Please fix.

File: collections/devops-tools/index.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ items:
  - komodorio/helm-dashboard
  - cloudquery/cloudquery
  - devtron-labs/devtron
+ - livecycle/preevy 
+
```
Comment (kenyonj) [inline collections/devops-tools/index.md:43]:
```suggestion
```

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**